### PR TITLE
Fix the missing example in HTTP rewrite

### PR DESCRIPTION
**Pull request by spacewander**
Also give them clearer names.
Signed-off-by: spacewander <spacewanderlzx@gmail.com>

**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

There is expected to be one more example in the rewrite section.

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: examples/standard/http-redirect-rewrite/httproute-rewrite-prefix-path.yaml [hunk 1]
```diff
@@ -8,7 +8,11 @@ spec:
   hostnames:
```
Comment (robscott) [inline examples/standard/http-redirect-rewrite/httproute-rewrite-prefix-path.yaml:8]: Can't leave a comment far enough up, but do you mind updating the "used in" section for this file? It looks like the current value is no longer used + the list does not contain the redirect and rewrite guide this is referenced in.

**Review by robscott — COMMENTED** [review]
Thanks @spacewander!

/ok-to-test
/approve

File: examples/standard/http-redirect-rewrite/httproute-rewrite-prefix-path.yaml (hunk 1 above)
Comment (spacewander) [reply examples/standard/http-redirect-rewrite/httproute-rewrite-prefix-path.yaml:8]: @robscott 
Done

**Comment by spacewander** [comment]
@robscott 
Would you review this PR again? Thanks!

**Comment by robscott** [comment]
Thanks @spacewander! Sorry I'd lost track of this one!

/lgtm

**Pull request merged**